app/views/subscriptions/index.slim

```diff
@@ -0,0 +1,159 @@
+.pricing-header.px-3.py-3.pt-md-5.pb-md-4.mx-auto.text-center style='max-width: 700px'
+  h1.display-4.text-monospace Time to choose
+  p.lead
+    ' Get access to premium features and help us continue improving the best speedrunning analytics system in the
+    ' universe 🌌
+.container
+  .card-deck.mb-3.text-center
+    - plan = current_user&.subscriptions&.active&.first
+    .card
+      .card-header
+        h4.my-0.font-weight-normal
+          ' Basic
+      .card-body
+        h1.card-title.pricing-card-title Free
+        ul.list-unstyled.mt-3.mb-4
+          li Basic analytics
+          li Ad-supported
+      .card-footer
+        .d-flex
+          button.btn.btn-lg.btn-outline-light.disabled.mr-2.flex-fill.tip(
+            style='cursor: default'
+            title="Active since #{current_user&.created_at&.to_date}"
+          ) Active
+          button.btn.btn-lg.btn-outline-light.tip-top.disabled title='This plan lasts forever.' style='cursor: default'
+            = icon('fas', 'infinity')
+    .card
+      .card-header
+        h4.my-0.font-weight-normal
+          ' Gold
+      .card-body
+        h1.card-title.pricing-card-title
+          ' $4
+          small.text-muted
+            ' / mo
+        ul.list-unstyled.mt-3.mb-4
+          li Disable ads
+          li Predict when you'll PB
+      .card-footer
+        - if plan && plan.stripe_plan_id == ENV['STRIPE_PLAN_ID_TIER1']
```

Comment:
This can be simplified to `plan&.stripe_plan_id == ENV['STRIPE_PLAN_ID_TIER1']` yeah?